evm/contracts/token-bridge/TokenBridge.sol

```diff
@@ -87,13 +86,15 @@ contract TokenBridge is LayerTransition{
         else{
             require(token.transfer(_recipient, _amountConverted), "TokenBridge: transfer failed");
         }
+        depositLimitRecord -= _amountConverted;
```

Comment:
wouldn't this be wrong?  If you refreshDepositLimit() it would be some rate that you transfer it out right?

Reply:
refreshDepositLimit is really just determining whether the 12 hour clock needs to be reset.